tests/jax/core_system_components/parameter_client_test.py

```diff
@@ -15,4 +15,410 @@
 
 """Tests for parameter client class for Jax-based Mava systems"""
 
-# TODO (Dries)
+import copy
+from types import SimpleNamespace
+from typing import Any, Dict, List, Sequence, Set, Union
+
+import jax
+import numpy as np
+import pytest
+
+from mava.callbacks.base import Callback
+from mava.systems.jax.parameter_client import ParameterClient
+from mava.systems.jax.parameter_server import ParameterServer
+
+
+class MockParameterServer(ParameterServer):
```

Comment:
Too late now, but it's for things like this where I wonder if we shouldn't have used monkeypatch